peripherals/wwdg/wwdg.yaml

```diff
@@ -0,0 +1,38 @@
+# The most common features, found on Basic timers TIM6 and TIM7,
+# and found on all more advanced timers.
+
+WWDG:
+  CR:
+    _modify:
+      WDGA: 
+        description: "Activation bit for the watchdog. This bit is set by software and only cleared after a reset."
+        bitOffset: 7
+        bitWidth: 1
+      T:
+        description: "These bits contain the value of the watchdog counter. A reset is produced when it rolls over from 0x40 to 0x3F (T6 us cleared)."
+        bitOffset: 0
+        bitWidth: 7
+  CFR:
+    WDGTB:
+      Div1: [0, "Counter clock (PCLK1 div 4096) div 1"]
+      Div2: [1, "Counter clock (PCLK1 div 4096) div 2"]
+      Div4: [2, "Counter clock (PCLK1 div 4096) div 4"]
+      Div8: [3, "Counter clock (PCLK1 div 4096) div 8"]
+      _merge:
+        - "WDGTB*"
```

Comment:
Merging is a good idea but the `_merge` key needs to belong to the `CFR`, one level up, not be part of the `WDGTB`. This should work ok if you just unindent it 2 lines (and perhaps put it before the WDGTB block for readability).